<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HintDialog</class>
 <widget class="QDialog" name="HintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(244, 244, 244)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最后剩余的单元格</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hintLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>260</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#8c8c8c</string>
   </property>
   <property name="text">
    <string>留意这个单元格及突出显示区域</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="progressLabel_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color:#a8a8a8</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="progressLabel_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color:#a8a8a8</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="rightToolButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
    border-radius: 25px;
    background-color: #EAEEF4;
    color: rgb(85, 170, 255);
}
QToolButton:hover {;
    background-color: #dce0e5;
}
QToolButton:pressed {;
    background-color: #cdd0d5;
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res/icons.qrc">
     <normaloff>:/res/arrow-right.png</normaloff>:/res/arrow-right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="leftToolButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
    border-radius: 25px;
    background-color: #EAEEF4;
    color: rgb(85, 170, 255);
}
QToolButton:hover {;
    background-color: #dce0e5;
}
QToolButton:pressed {;
    background-color: #cdd0d5;
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res/icons.qrc">
     <normaloff>:/res/arrow-left.png</normaloff>:/res/arrow-left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="okToolButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
    border-radius: 25px;
    background-color: #EAEEF4;
    color: rgb(85, 170, 255);
}
QToolButton:hover {;
    background-color: #dce0e5;
}
QToolButton:pressed {;
    background-color: #cdd0d5;
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res/icons.qrc">
     <normaloff>:/res/ok.png</normaloff>:/res/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
